.changeset/breezy-eyes-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": patch
+---
+
+Fix history navigation to hash fragment on Firefox when using view transitions
```

Comment:
```suggestion
Fixes history navigation to hash fragment on Firefox when using View Transitions
```

Reply:
I thought we were deliberately using the imperative form?

Reply:
I thought the opposite 🤔 we need Sarah (on phone rn, I can't ping for some reason)

Reply:
@sarah11918 to the rescue!